src/sys/bindings/types.rs

```diff
@@ -146,6 +146,7 @@ pub enum KeyConversion {
 }
 
 #[cfg(feature = "napi-4")]
+#[allow(dead_code)]
```

Comment:
It's necessary because we don't use both variants. For completeness, it's better to keep both. We can remove this if we implement bounded queues/channels.